admin/services/pupil-access-arrangements-edit.service.js

```diff
@@ -0,0 +1,29 @@
+const R = require('ramda')
+const pupilAccessArrangementsService = require('../services/pupil-access-arrangements.service')
+const pupilService = require('../services/pupil.service')
+const monitor = require('../helpers/monitor')
+
+const pupilAccessArrangementsEditService = {}
+
+/**
+ * Get pupil access arrangements edit data
+ * @returns {Promise<Array>}
+ */
+pupilAccessArrangementsEditService.getEditData = async (requestData, pupilUrlSlug, dfeNumber) => {
+  if (!pupilUrlSlug) {
+    throw new Error('Pupil url slug not provided')
+  }
+  const reqBody = R.clone(requestData)
```

Comment:
are you passing in a raw http request?  if so, i would consider parsing that in the controller and declare more explicit parameters on the service method

Reply:
@GuyHarwood updated so on both submission and editing controller areas an explicit object is being passed down to the corresponding services.